src/templates/console.html

```diff
@@ -234,7 +234,22 @@
           $(".cmd-cursor").addClass("noblink");
         }
       }
-      window.console_ready = main();
+      // Dynamically update the indexURL to use with Pyodide
+      var script = document.createElement("script");
+      script.addEventListener("load", function () {
+        window.console_ready = main();
+      });
+
+      let indexURL = "{{ PYODIDE_BASE_URL }}";
+      const urlParams = new URLSearchParams(window.location.search);
+      if (urlParams.has("build")) {
+        indexURL = indexURL.replace(
+          "/full/",
+          "/" + urlParams.get("build") + "/",
+        );
+      }
+      script.src = indexURL + "pyodide.js";
+      document.head.appendChild(script);
```

Comment:
Rather than this script tag stuff, you should use `await import(script);`. This can go in the top of main.

Reply:
Yes, good idea, done. Also added some validation for the URL param to be in ['full', 'debug', 'pyc'] for now.